scene/main/viewport.h

```diff
@@ -492,10 +492,10 @@ class Viewport : public Node {
 	void _window_start_resize(SubWindowResize p_edge, Window *p_window);
 
 protected:
-	bool _set_size(const Size2i &p_size, const Size2i &p_size_2d_override, bool p_allocated);
+	bool _set_size(const Size2i &p_size, const Size2 &p_size_2d_override, bool p_allocated);
 
 	Size2i _get_size() const;
-	Size2i _get_size_2d_override() const;
+	Size2 _get_size_2d_override() const;
```

Comment:
Suggesting reversion. The MRP is fixed without `size_2d_override` being a `Size2`.

```suggestion
	Size2i _get_size_2d_override() const;
```